.github/workflows/stale-prs.yml

```diff
@@ -0,0 +1,151 @@
+---
+name: "Close Stale PRs"
+
+on:
+  schedule:
+    # Run daily at 00:00 UTC
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+permissions:
+  pull-requests: write
+  issues: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Warn and close stale PRs
+        uses: actions/github-script@ed597411d8f924073f98dfc5c65a23a2325f34cd 
+        with:
+          script: |
+            const daysUntilStale = 46;  // First warning: 2 weeks before 60-day deadline
+            const daysUntilSecondWarning = 53;  // Second warning: 1 week before 60-day deadline
+            const daysUntilClose = 60;  // Close PRs after 60 days of inactivity
+            
+            const exemptLabels = ['pinned', 'security', 'dependencies'];
```

Comment:
I'd be tempted to have a single "do not autoclose" label and manually apply it, if anything.